src/blockdata/script.rs

```diff
@@ -632,113 +632,79 @@ pub enum Instruction<'a> {
 
 /// Iterator over a script returning parsed opcodes
 pub struct Instructions<'a> {
-    data: &'a [u8],
+    data: ::core::slice::Iter<'a, u8>,
     enforce_minimal: bool,
 }
 
+impl<'a> Instructions<'a> {
+    fn next_push_data_len(&mut self, len: usize, max: usize) -> Option<Result<Instruction<'a>, Error>> {
```

Comment:
In b5d5e6310e1767bd68be27dd5ed2c7f89d3bfab0:

I think it makes more sense for `max` to be called `min` or even `min_n`.